scripts/run-ci.sh

```diff
@@ -46,6 +46,10 @@ publishToLocalRegistry
 
 yarn bootstrap:examples
 
+# Get travis's chrome version and download the appropriate webdriver-manager for protractor
+CHROME_VERSION=`google-chrome --version | egrep -o '[0-9.]+' | head -1`
+npx lerna exec --ignore '@angular-builders/*' -- ./node_modules/protractor/bin/webdriver-manager update --versions.chrome $CHROME_VERSION
```

Comment:
What is the reason for this?

Reply:
I cherry picked this from #886. It was to skip resolved errors. Now that #886 is merged, I rebase this pull request onto master and remove cherry picked commits
Thanks for the question